devices.js

```diff
@@ -6409,6 +6409,72 @@ const devices = [
         fromZigbee: [fz.on_off],
         toZigbee: [tz.on_off],
     },
+
+    // Drayton Wiser
+    {
+        zigbeeModel: ['iTRV'],
+        model: 'WV704R0A0902',
+        vendor: 'Dryton Wiser',
+        description: 'Smart heating thermostatic radiator valve',
+        supports: 'temperature, battery, keypad lock, heating demand',
+        fromZigbee: [
+            fz.ignore_basic_report,
+            fz.ignore_haDiagnostic,
+            fz.ignore_genOta,
+            fz.ignore_zclversion_read,
+            fz.drayton_thermostat,
+            fz.drayton_itrv_battery,
+            fz.drayton_user_interface,
+            fz.drayton_device_info,
+        ],
+        toZigbee: [
+            tz.thermostat_occupied_heating_setpoint,
+            tz.thermostat_keypad_lockout,
+        ],
+        meta: {configureKey: 1},
+        configure: async (device, coordinatorEndpoint) => {
+            const endpoint = device.getEndpoint(1);
+            const binds = [
+                'genBasic', 'genPowerCfg', 'genIdentify', 'genPollCtrl',
+                'hvacThermostat', 'hvacUserInterfaceCfg', 'haDiagnostic',
+            ];
+            await bind(endpoint, coordinatorEndpoint, binds);
+            await configureReporting.batteryVoltage(endpoint);
+            await configureReporting.thermostatTemperature(endpoint);
```

Comment:
Do we perhaps also want to query this every minute like the other values?

```await configureReporting.thermostatTemperature(endpoint, repInterval.MINUTE, repInterval.MAX);```

Although we might want to set the max reporting to repInterval.HOUR for all of these instead of repInterval.MAX, repInterval.MAX is a very long time!

Reply:
It is not clear how the min and max work, most examples use MAX for the maximumReportInterval so I went with that... hehe

Reply:
I spend a few days digging into this for the Eurotronic's unit.

min=report no more than
max=report ATLEAST this often

min=5,max=10 -> report every 10 minutes, but you are allowed to report after 5 minutes (e.g. if the value is changing a lot)

MAX=62000 (https://github.com/Koenkk/zigbee-herdsman-converters/blob/master/devices.js#L9) which is a long time

I think hour would be better, or maybe 15 or even 10 minutes (I added a value for 10 minutes for the eurotronic device but the PR is not merged yet) e.g. https://github.com/Koenkk/zigbee-herdsman-converters/pull/850/files#diff-6c9a6acf22f90d1c6e524d9f3c5c1745R11